.github/workflows/ci-cd-release.yml

```diff
@@ -20,6 +20,14 @@ jobs:
         uses: actions/setup-node@v1
         with:
           node-version: ${{ matrix.node-version }}
+      - name: Install, build, test, lint
+        run: |
+          npm i
```

Comment:
the order:

1. npm i
2. npm run lint
3. npm run build
4. npm run test

AND

pls create a separate script for these: e.g. `assemble` that includes all the scripts above, so that we won't specify those scripts everywhere.

Reply:
done